src/main/java/com/crud/library/controller/ExemplarController.java

```diff
@@ -0,0 +1,30 @@
+package com.crud.library.controller;
+
+import com.crud.library.dto.ExemplarDto;
+import com.crud.library.service.ExemplarService;
+import com.crud.library.status.RentalStatus;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/v1/exemplar")
+public class ExemplarController {
+
+    private final ExemplarService service;
+
+    @GetMapping(value = "/{id}")
+    public long getAvailable(@PathVariable long id) {
```

Comment:
Czy aby na pewno? Co to jest to id? ;)

Reply:
to jest book ID, wyszukuje wszystkie dostępne egzemplarze danej książki, zmieniam na @RequestParam